components/HackneyDS/Conversation/style.scss

```diff
@@ -0,0 +1,170 @@
+.lbh-conversation__metadata {
+  margin-top: 10px;
+}
+
+.govuk-radios__input:checked + .govuk-radios__label:after,
```

Comment:
'.govuk-radios__input:checked + .govuk-radios__label:after' does not belong in here, please delete

Reply:
done